components/reactions/storedproc-mssql/src/config.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2025 The Drasi Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Configuration for the MS SQL Server Stored Procedure reaction.
+
+use serde::{Deserialize, Serialize};
+
+/// Configuration for the MS SQL Server Stored Procedure reaction
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct MsSqlStoredProcReactionConfig {
+    /// Database hostname or IP address
+    #[serde(default = "default_hostname")]
+    pub hostname: String,
+
+    /// Database port (default: 1433)
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub port: Option<u16>,
+
+    /// Database user
+    pub user: String,
+
+    /// Database password
+    pub password: String,
```

Comment:
How are we thinking about supporting other forms of auth, such as managed identities, etc...?